packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -34,6 +34,7 @@ export interface AssetsOnlyConfig extends SanitizedWorkerConfig {
 	topLevelName: Defined<SanitizedWorkerConfig["topLevelName"]>;
 	name: Defined<SanitizedWorkerConfig["name"]>;
 	compatibility_date: Defined<SanitizedWorkerConfig["compatibility_date"]>;
+	compatibility_flags: Defined<SanitizedWorkerConfig["compatibility_flags"]>;
```

Comment:
Don't think this will be necessary as it's always defined in the returned config.

```suggestion
```